client/blocks/plugins-update-manager/schedule-create.tsx

```diff
@@ -31,7 +33,7 @@ export const ScheduleCreate = ( props: Props ) => {
 				<div className="ch-placeholder"></div>
 			</CardHeader>
 			<CardBody>
-				<ScheduleForm />
+				<ScheduleForm siteSlug={ siteSlug } />
```

Comment:
Is there a reason we need to pass `siteSlug` all the way down the tree and `ScheduleForm` can't grab it itself from state?